scripts/binary/binary-integrity-check-source.js

```diff
@@ -0,0 +1,168 @@
+const OrigError = Error
+const captureStackTrace = Error.captureStackTrace
+const toString = Function.prototype.toString
+const callFn = Function.call
+
+// eslint-disable-next-line no-unused-vars
+const stackIntegrityCheck = function stackIntegrityCheck (options) {
+  const originalStackTraceLimit = OrigError.stackTraceLimit
+  const originalStackTrace = OrigError.prepareStackTrace
+
+  OrigError.stackTraceLimit = Infinity
+
+  OrigError.prepareStackTrace = function (_, stack) {
+    return stack
+  }
+
+  const tempError = new OrigError
```

Comment:
Is is possible clone the class instance so we don't need to reset the overrides?

Reply:
I think there's some alternative things we can do here, but I think I like this pattern best.